ch02/probability_intro.md

```diff
@@ -0,0 +1,51 @@
+
+An **event** is any subset of the sample space, to which a probability can be assigned. For example, "getting one tails and one heads" would be an event for the coin toss experiment mentioned above. The outcomes that make up the event are $\{HT, TH\}$ and the probability of this event occuring is $\frac{1}{2}$.
+
+## Probability of Events
+
+Let us consider a sample space where **every outcome is equally likely to occur**. Then, the probability of an event occurring can be calculated as the ratio of the number of outcomes in the event to the total number of events in the sample space. $ P(\text{an event happens}) = \frac{\text{# of outcomes that make the event happen}}{\text{# of total outcomes}} $.    
+In the coin toss experiment, the events $\{HH, HT, TH, TT\}$ are all equally likely to occur, so to calculate the probability of getting one tails and one heads, we can count the number of outcomes in the event (2) and divide by the total number of outcomes (4) : $P(\text{one heads and one tails}) = \frac{2}{4} = \frac{1}{2} $
+
+There are a few important properties of probabilities of events:
+
+1. The probability of an event is between 0 and 1: $0 \leq P(E) \leq 1$
+2. The probabilities of all the outcomes in a sample space sum to 1: $\sum_{A \in \Omega} P(A) = 1$
+
+## Addition Rule
+Sometimes, we want to calculate the probabilities involving one event OR another event occuring. To do so, we add their probabilities.
+
+When two events $A$ and $B$ are disjoint (they don't share any common outcomes), then the probability that events $A$ or $B$ ($A \cup B$) occur is the sum of the probability of each event. $P(A \cup B) = P(A) + P(B) \text{ if } A \cap B = \varnothing$. For instance, the events "getting two heads" and "getting two tails" with two coin tosses are disjoint. Therefore, the probability of the combined event "getting two heads or two tails" is equal to the sum of the individual events. $P(\text{getting two heads or two tails}) = P(\text{getting two heads}) + P(\text{getting two tails}) = \frac{1}{4} + \frac{1}{4} = \frac{1}{2}$
+
+When two events $A$ and $B$ aren't disjoint (they share common outcomes), then the probability that $A$ or $B$ ($A \cup B$) occur is the sum of the probability of each event, minus the probability of both events occurring ($A \cap B$). $P(A \cup B) = P(A) + P(B) - P(A \cap B) \text{ if } A \cap B \neq \varnothing$. The intuition behind this can better be visualized through a venn diagram.
+
+![probability_intro_venn](../assets/probability_intro_venn.png)
+
+When we have two events $A$ and $B$ that have common outcomes and we want to find the probability that either event happens, we want the area inside the circles. However, if we add together the probabilities of $A$ occurring and $B$ occurring, we count the intersection of the circles, $ P(A \cap B) $, twice. That's why we need so subtract the intersection once.
```

Comment:
First sentence: "When we have two events that aren't disjoint, we want to find the area inside the circles to compute the probability that either event happens."

Reply:
fixed